src/mach/mod.rs

```diff
@@ -51,6 +73,25 @@ pub struct MachO<'a> {
     bind_interpreter: Option<imports::BindInterpreter<'a>>,
 }
 
+#[cfg(feature = "std")]
+impl<'a> fmt::Debug for MachO<'a> {
+    fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
+        fmt.debug_struct("MachO")
+            .field("header",        &self.header)
+            .field("load_commands", &self.load_commands)
+            .field("segments",      &self.segments)
+            .field("entry",         &self.entry)
+            .field("libs",          &self.libs)
+            .field("name",          &self.name)
+            .field("little_endian", &self.little_endian)
+            .field("is_64",         &self.is_64)
+            .field("symbols",       &self.symbols().collect::<Vec<_>>())
+            .field("exports",       &self.exports().unwrap())
```

Comment:
Hmm… these don't need to be `unwrap()` at all. It's `Debug` output; how would you feel about printing `Ok(…)` and `Err(…)` here instead of returning an error?